src/main/java/gov/cms/madie/models/measure/Measure.java

```diff
@@ -113,22 +115,23 @@ public class Measure extends ResourceAcl {
   private List<DefDescPair> riskAdjustments;
   private ProgramUseContext programUseContext;
 
+  @NotBlank(message = "Model required")
```

Comment:
maybe "Model is required."

Reply:
@adongare Just changed it, thanks Ashok!